ticketbookingapp/src/main/java/pl/touk/recruitmenttask/ticketbookingapp/service/converter/TicketTypeAttributeConverter.java

```diff
@@ -0,0 +1,42 @@
+package pl.touk.recruitmenttask.ticketbookingapp.service.converter;
+
+import pl.touk.recruitmenttask.ticketbookingapp.model.TicketType;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+
+@Converter
+public class TicketTypeAttributeConverter implements AttributeConverter<TicketType, Integer> {
+
+    @Override
+    public Integer convertToDatabaseColumn(TicketType attribute) {
```

Comment:
dlaczego trzymamy enumy jako liczby?

Reply:
Niesie to za sobą plusy i minusy. Zamiast trzymania enum'ów jako liczby w bazie, moglibyśmy je trzymać jako string'i.
Plusy: 
- string'i zajmują więcej miejsca w bazie
- odczyt string'ów z bazy jest wolniejszy od odczytu int'ów
- przy refaktorze enum'ów zachowujemy spójność z bazą danych

Minusy:
- mała czytelność w bazie danych